x/committee/client/cli/tx.go

```diff
@@ -0,0 +1,235 @@
+			proposalID, err := strconv.ParseUint(args[0], 10, 64)
+			if err != nil {
+				return fmt.Errorf("proposal-id %s not a valid int, please input a valid proposal-id", args[0])
+			}
+
+			// Build vote message and run basic validation
+			msg := types.NewMsgVote(from, proposalID)
+			err = msg.ValidateBasic()
+			if err != nil {
+				return err
+			}
+
+			return utils.GenerateOrBroadcastMsgs(cliCtx, txBldr, []sdk.Msg{msg})
+		},
+	}
+}
+
+// GetGovCmdSubmitProposal returns a command to submit a proposal to the gov module. It is passed to the gov module for use on its command subtree.
+func GetGovCmdSubmitProposal(cdc *codec.Codec) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "committee [proposal-file] [deposit]",
+		Short: "Submit a governance proposal to change a committee.",
+		Long: fmt.Sprintf(`Submit a governance proposal to create, alter, or delete a committee.
+
+The proposal file must be the json encoded form of the proposal type you want to submit.
+For example, to create or update a committee:
+%s
+
+and to delete a committee:
+%s
+`, mustGetExampleCommitteeChangeProposal(cdc), mustGetExampleCommitteeDeleteProposal(cdc)),
+		Args: cobra.ExactArgs(2),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			txBldr := auth.NewTxBuilderFromCLI().WithTxEncoder(utils.GetTxEncoder(cdc))
+			cliCtx := context.NewCLIContext().WithCodec(cdc)
+
+			// Get proposing address
+			proposer := cliCtx.GetFromAddress()
+
+			// Get the deposit
+			deposit, err := sdk.ParseCoins(args[1])
+			if err != nil {
+				return err
+			}
+
+			// Get the proposal
+			bz, err := ioutil.ReadFile(args[0])
+			if err != nil {
+				return err
+			}
+			var content govtypes.Content
+			if err := cdc.UnmarshalJSON(bz, &content); err != nil {
+				return err
+			}
+			if err = content.ValidateBasic(); err != nil {
+				return err
+			}
+
+			// Build message and run basic validation
+			msg := govtypes.NewMsgSubmitProposal(content, deposit, proposer)
+			err = msg.ValidateBasic()
+			if err != nil {
+				return err
+			}
+
+			// Sign and broadcast message
+			return utils.GenerateOrBroadcastMsgs(cliCtx, txBldr, []sdk.Msg{msg})
+		},
+	}
+	return cmd
+}
+
+// mustGetExampleCommitteeChangeProposal is a helper function to return an example json proposal
+func mustGetExampleCommitteeChangeProposal(cdc *codec.Codec) string {
+	exampleChangeProposal := types.NewCommitteeChangeProposal(
+		"A Title",
+		"A description of this proposal.",
+		types.NewCommittee(
+			1,
+			"The description of this committee.",
+			[]sdk.AccAddress{sdk.AccAddress(crypto.AddressHash([]byte("exampleAddres")))},
+			[]types.Permission{
+				types.ParamChangePermission{
+					AllowedParams: types.AllowedParams{{Subspace: "cdp", Key: "CircuitBreaker"}},
+				},
+			},
+			sdk.MustNewDecFromStr("0.8"),
+			time.Hour*24*7,
+		),
+	)
+	exampleChangeProposalBz, err := cdc.MarshalJSONIndent(exampleChangeProposal, "", "  ")
+	if err != nil {
+		panic(err)
+	}
```

Comment:
The Long field value is evaluated when the cmd struct is created, which happens when all the commands are created in kvcli main.go. So if there's a problem it should panic when any command runs.
But it would nicer if problems were caught sooner, so I'll add a test.